src/transformStateWithClones.js

```diff
@@ -65,7 +65,33 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, transforms) {
-  // write code here
+  let currentArray = Object.assign({}, state);
+
+  for (let i = 0; i < transforms.length; i++) {
+    switch (transforms[i].operation) {
```

Comment:
```suggestion
    switch (transforms.operation) {
```
use for..of instead of usual for loop